Sources/RidiOAuth2/AuthorizationError.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+public struct AuthorizationError: Swift.Error {
+    var underlyingError: Swift.Error?
+    var statusCode: Int?
+    var authorizationErrorCode: String?
+    var authorizationErrorDescription: String?
```

Comment:
struct `AuthorizationError`가 오류와 관련되어 있다는 맥락을 전달하고 있기 때문에 생략시키고, 보다 (`CustomNSError`와의 중복을 피하는 목적으론) 명확한 의미를 가진 네이밍을 사용할 수 있겠어요.
```suggestion
    var apiErrorCode: String?
    var apiErrordescription: String?
```

Reply:
👍

Reply:
00c7ff84 수정 완료